adhoc/us-supreme-court-cases/import-transcripts.py

```diff
@@ -152,3 +144,15 @@ def convert_lists(data, cols):
     convert_dates(case, date_convert)
     
     cases.append(case)    
+
+cases_df       = pandas.DataFrame(cases)
+transcripts_df = pandas.DataFrame(transcripts) 
+
+transcripts_pks = ['case_name', 'title', 'speaker', 'start']
+
+repo = Dolt('./')
+repo.import_df('cases', cases_df, ['case_name'], import_mode='replace')
+repo.import_df('transcripts',
+               transcripts_df,
+               transcripts_pks,
+               import_mode='replace')
```

Comment:
Don't you need to clone the remote and then push to it? Doltpy can take care of that.

Reply:
I wanted to examine the diffs, etc. As I said, this is a one off script.